docs/sdk/Cargo.toml

```
@@ -17,7 +17,7 @@ workspace = true
 # Needed for all FRAME-based code
 parity-scale-codec = { version = "3.0.0", default-features = false }
 scale-info = { version = "2.6.0", default-features = false }
-frame = { path = "../../substrate/frame", features = [
+polkadot-sdk-frame = { path = "../../substrate/frame", features = [
```

```suggestion
frame = { package = "polkadot-sdk-frame", path = "../../substrate/frame", features = [
```
Then you dont have to change the code.

---

Yeah, @rzadp tried it in the linked issue https://github.com/paritytech/eng-automation/issues/6#issuecomment-2012428997 but it didn’t work for the released crate.